ios/TPSStripe/TPSCardField.m

```diff
@@ -179,4 +179,18 @@ - (void)paymentCardTextFieldDidChange:(STPPaymentCardTextField *)textField {
                 });
 }
 
+- (void)paymentCardTextFieldDidEndEditing:(STPPaymentCardTextField *)textField {
```

Comment:
It took some digging but I discovered the availability of this callback by examining the source of the module that this extends from `STPPaymentCardTextField`. Per [this line](https://github.com/stripe/stripe-ios/blob/master/Stripe/STPPaymentCardTextField.m#L1440) a module that inherits(?)/extends(?) from this can set `paymentCardTextFieldDidEndEditing` in order to be notified when the user finishes editing the combined fields.